docs/source/configuration/configuration_basics.md

```diff
@@ -241,3 +241,21 @@ Customise the configuration loader arguments in `settings.py` as follows if your
 ```python
 CONFIG_LOADER_ARGS = {"default_run_env": "base"}
 ```
+
+### How to use Kedro without the rich library
```

Comment:
I think this section will fit better in the logging docs: https://docs.kedro.org/en/stable/logging/index.html

Reply:
I thought about putting it in the logging page, since it already mentions `rich` a lot, but since it affects features other than logging I thought it'd fit better as more general configuration information. What do you think about adding it to both pages?